docs/manifest.json

```diff
@@ -11,6 +11,12 @@
 		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/language.md",
 		"parent": null
 	},
+	{
+		"title": "Extensibility",
```

Comment:
Might be good to move "theme support" below this category.

Reply:
Agreed, Can we wait until we have other Extensibility "subcategories" like a page dedicated to "filters"...?